File: src/main/java/com/back/domain/game/game/repository/PlatformRepository.java
package com.back.domain.game.game.repository;

import com.back.domain.game.game.entity.Platform;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface PlatformRepository extends JpaRepository<Platform, Long> {
    List<Platform> findByIgdbIdIn(List<Long> igdbIds);
}

File: src/main/kotlin/com/back/domain/game/game/repository/PlatformRepository.kt
package com.back.domain.game.game.repository

import com.back.domain.game.game.entity.Platform
import org.springframework.data.jpa.repository.JpaRepository
import java.util.Optional

interface PlatformRepository : JpaRepository<Platform, Long> {
    fun findByIgdbId(igdbId: Long): Optional<Platform>

    fun findByIgdbIdIn(igdbIds: List<Long>): List<Platform>
}
